apps/studio/src/features/editing-experience/components/utils.ts

```diff
@@ -91,3 +107,35 @@ export const generatePreviewSitemap = (
     })),
   } as IsomerGeneratedSiteProps["siteMap"]
 }
+
+export const getIframeSrc = (embedCode: string) => {
+  const elem = DOMPurify.sanitize(embedCode, {
+    ALLOWED_TAGS: ["iframe"],
+    RETURN_DOM_FRAGMENT: true,
+  })
+  const sanitizedUrl = elem.firstElementChild?.getAttribute("src")
+
+  if (sanitizedUrl === null) {
+    return undefined
+  }
+
+  return sanitizedUrl
+}
```

Comment:
- explicitly state the return types so its upfront to the future user of this method 
- simplify the return statements

```suggestion
export const getIframeSrc = (embedCode: string): string | undefined => {
  const elem = DOMPurify.sanitize(embedCode, {
    ALLOWED_TAGS: ["iframe"],
    RETURN_DOM_FRAGMENT: true,
  })
  const sanitizedUrl = elem.firstElementChild?.getAttribute("src")
  return sanitizedUrl ?? undefined
}
```

Reply:
Updated in bac88de4fe3d5e3260e22751ad78b8a3c56a9213